Cargo.lock

```diff
@@ -365,42 +357,40 @@ dependencies = [
 
 [[package]]
 name = "clap"
-version = "4.3.3"
+version = "4.4.2"
```

Comment:
Could you please, update the lock file only by the means of added dependencies? So you actually won't have broken tests due to clap change? It'll also be easier for us to test things and see what was added.

You could though, remove dups if you see a way to do so.